internal/flags/activate.go

```diff
@@ -64,6 +65,15 @@ func (f *Flags) handleActivateCommand() utils.ReturnCode {
 			f.amtActivateCommand.Usage()
 			return utils.MissingOrIncorrectProfile
 		}
+		if f.UUID != "" {
+			uuidPattern := `[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`
```

Comment:
use `uuidPattern := regexp.MustCompile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$")` to ensure the regex compile and is correct, i also added starts and endwith markers. then use `uuidPattern.MatchString(f.UUID)`